source/FreeRTOS_Sockets.c

```diff
@@ -704,15 +704,18 @@ Socket_t FreeRTOS_socket( BaseType_t xDomain,
 
 /**
  * @brief The select() statement: wait for an event to occur on any of the sockets
- *        included in a socket set.
+ *        included in a socket set and return its event bits when the event occurs.
  *
  * @param[in] xSocketSet: The socket set including the sockets on which we are
  *                        waiting for an event to occur.
  * @param[in] xBlockTimeTicks: Maximum time ticks to wait for an event to occur.
  *                   If the value is 'portMAX_DELAY' then the function will wait
  *                   indefinitely for an event to occur.
  *
- * @return The socket which might have triggered the event bit.
+ * @return The event bits (event flags) value for the socket set in which an
+ *          event occurred. During the call if any socket is signalled
+ *          (using FreeRTOS_SignalSocket), then eSELECT_INTR is returned.
```

Comment:
I would suggest the following change:
~~~diff
-    During the call if any socket is signalled
-    (using FreeRTOS_SignalSocket), then eSELECT_INTR is returned.

+    If any socket is signalled during the call, using FreeRTOS_SignalSocket()
+    or FreeRTOS_SignalSocketFromISR(), then eSELECT_INTR is returned.
~~~

Reply:
updated